src/Views/General.vala

```diff
@@ -82,20 +82,18 @@ public class MouseTouchpad.GeneralView : Gtk.Grid {
         hold_switch.halign = Gtk.Align.START;
 
         var hold_help = new Gtk.Label (_("Long-pressing and releasing the primary button will secondary click."));
-        hold_help.margin_bottom = 6;
+        hold_help.margin_bottom = 18;
         hold_help.wrap = true;
         hold_help.xalign = 0;
         hold_help.get_style_context ().add_class (Gtk.STYLE_CLASS_DIM_LABEL);
 
-        var hold_length_label = new SettingLabel (_("Long-press length:"));
-        hold_length_label.margin_bottom = 18;
+        var hold_length_label = new SettingLabel (_("Length:"));
 
         var hold_scale = new Gtk.Scale.with_range (Gtk.Orientation.HORIZONTAL, 0.5, 2.0, 0.1);
         hold_scale.draw_value = false;
         hold_scale.hexpand = true;
-        hold_scale.margin_bottom = 10;
         hold_scale.width_request = 160;
-        hold_scale.add_mark (1.2, Gtk.PositionType.BOTTOM, null);
```

Comment:
Why move the markers to top for these inline scales? This seems inconsistent with other pages?

Reply:
It lines the scale up better with the baseline of the text; I'm open to a different way to do that, though.

Reply:
Fair point. Just for reference: 
![screenshot from 2019-01-08 01 39 13](https://user-images.githubusercontent.com/523210/50802198-c2b60e80-12e6-11e9-8a30-329671e54274.png)
![screenshot from 2019-01-08 01 38 45](https://user-images.githubusercontent.com/523210/50802200-c2b60e80-12e6-11e9-90f7-4e2cd4d9cc84.png)

But for such a small difference I wouldn't change the consistency? Maybe the alignment with text of the slider could be improved in general later on? If I for example add both TOP and BOTTOM it's nicely vertically centered with the text.